exercises/practice/wordy/.meta/config.json

```diff
@@ -0,0 +1,20 @@
+{
+  "authors": ["Ephraim-nonso"],
```

Comment:
Open your terminal at the root directory and run:
```console
./bin/format_all.sh
```

Reply:
Hi. Concerning this comment, I'm unable to run this script because my current repository do not have this exact file in it. When I navigated into the bin folder, here are the items I have found: